### PR TITLE
Th header component

### PR DESCRIPTION
**Pull request by relyt4me**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features

### Detailed Description / What does this PR do?
Creates the Header and Search area as well as complete merge conflicts on basis of style.

### Has This Been Tested?
- [ ] No
- [ ] Yes

### What are the relevant tickets?
Header folder
Search folder
css for all files
background image

### Message/Questions for reviewer:
The Beon font is no longer working and may need to be researched more

### Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by tarynmartin — APPROVED** [review]
This looks great! We should add some error handling tests for search in the future.

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+}
+
+.logo {
+  height: 12vh;
+  grid-column: 3/4;
+  border-radius: 45%;
+}
+.logo:hover {
```
Comment (rwilliams659) [inline src/components/Header/Header.css:16]: Love the hover effects!

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import f2glogo from '../../assets/f2glogo.png';
+import './Header.css';
+
+const Header = () => {
+  return (
+    <header className='header-component'>
+      <h1 className='header-title'>Fridge to Glass</h1>
+      <img src={f2glogo} alt='Three neon glasses logo for Fridge to Glass' className='logo' />
```
Comment (rwilliams659) [inline src/components/Header/Header.js:9]: You probably want to just change the alt text to "Fridge to Glass" - I think we were getting a warning in the console not to use the word "logo" in alt text

**Review by rwilliams659 — APPROVED** [review]
Looks great! Just had the 1 comment re alt text

**Pull request merged**